docs/PAYLOAD.md

```diff
@@ -2024,6 +2025,35 @@ For some users of the plugin they are unable to get messages sent via GCM to sho
 }
 ```
 
+# FCM Payload Details
+
+Here's a sample JSON payload to send a push notification to an Android or iOS device using the FCM app server protocol:
+
+```php
```

Comment:
I think this could be `json` instead of `php`.

Reply:
Fixed in https://github.com/phonegap/phonegap-plugin-push/commit/fa2434af77763e9534229f65c99b19f47d592f71#diff-41648f6c35415f345028d85fb4dd3b02L2032